contracts/test/StakeTable.t.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: Unlicensed
+
+/* solhint-disable contract-name-camelcase, func-name-mixedcase, one-contract-per-file */
+
+pragma solidity ^0.8.0;
+
+// Libraries
+import "forge-std/Test.sol";
+import { BN254 } from "bn254/BN254.sol";
+import { BLSSig } from "../src/libraries/BLSSig.sol";
+import { EdOnBN254 } from "../src/libraries/EdOnBn254.sol";
+import { IStakeTable } from "../src/interfaces/IStakeTable.sol";
+import { LightClient } from "../src/LightClient.sol";
+
+// Token contract
+import { ExampleToken } from "../src/ExampleToken.sol";
+
+// Target contract
+import { StakeTable as S } from "../src/StakeTable.sol";
+
+contract StakeTable_register_Test is Test {
+    event Registered(bytes32, uint64, IStakeTable.StakeType, uint256);
+
+    S public stakeTable;
+    ExampleToken public token;
+    LightClient public lightClientContract;
+    address public lightClientAddress;
+    address public tokenAddress;
+    address exampleTokenCreator;
+    uint256 constant INITIAL_BALANCE = 1_000;
+    uint64 depositAmount;
+    uint64 validUntilEpoch;
+
+    function computeGroupElements(address sender)
```

Comment:
rename to `prepareWallet`? or some names like that?

Reply:
Yes, renamed to `genClientWallet` in df906e7.